packages/syft/src/syft/service/notification/email_templates.py

```diff
@@ -50,6 +50,28 @@ def email_body(notification: "Notification", context: AuthedServiceContext) -> s
         if result.is_err():
             raise Exception("Couldn't update the user password")
 
+        # Get the token expiry time from settings
+        # Option 1
+        # relative
+        # from ..settings.settings_service import SettingsService
+
+        # method = context.server.get_service_method(SettingsService.get)
+        # settings = method(context=context)
+        # expiry_time = settings.pwd_token_config.token_exp_min
+
+        # Option 2
+        settings_service = context.server.get_service("settingsservice")
```

Comment:
should we update to the context.server.services approach and remove the rest of this comment?

Reply:
Good point, will merge it with Teo's PR.